srcpkgs/wcm/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'wcm'
 pkgname=wcm
-version=0.7.0
+version=0.7.5
 revision=1
 build_style=meson
+configure_args="-Denable_wdisplays=false"
```

Comment:
Out of curiosity, why disable this?

Reply:
Enabling that option (as is the default) causes it to build wdisplays as a submodule from an arbitrary git commit in a fork that is not Void's upstream for that project. It's a new option (or at least newer than the previous release) and I can see why the developer included it (no canonical source for wdisplays after the original developer deleted their repo), but it would just cause problems for us.